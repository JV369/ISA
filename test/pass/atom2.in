./feedreader http://www.mocky.io/v2/5bcdec782f00005200c8564c -aTu
