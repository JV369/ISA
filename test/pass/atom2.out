*** Test ***
Test entry
Autor: JV
Aktualizace: 2003-12-13T18:30:02Z
URL: http://hue.org

Test entry 2
Autor: JÃ¶rvearick
Aktualizace: 2003-12-13T18:30:02Z
URL: http://hue.com
